VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/monitoring-graph/monitoring-graph.component.scss

```diff
@@ -60,3 +60,79 @@
   width: 100%;
   margin-top: 30px;
 }
+
+.box {
+  background: #fff;
+  width: auto;
+  height: auto;
+  border-radius: 3px;
+  position: absolute;
+  z-index: 99;
+  box-shadow: 0 7px 11px 0px rgba(0, 0, 0, 0.3);
+}
+.boxHead {
+  font-family: 'GDS Transport', Arial, sans-serif;
```

Comment:
Could you stick to the GovUK styles i.e. 
@include govuk-font($size: 16, $weight: 'bold');

Reply:
replaced with mixins